### PR TITLE
build next app for testing

### PR DESCRIPTION
**Pull request by ryanjduffy**

### CODE REVIEW

File: .github/actions/test/main.js [hunk 1]
```diff
@@ -40,43 +36,6 @@ spawnChecked("chmod", ["+x", "replay-node/macOS-replay-node"]);
 // Set environment variables needed to replay node recordings.
 process.env.RECORD_REPLAY_NODE = "replay-node/macOS-replay-node";
 process.env.RECORD_REPLAY_DRIVER = "replay-driver/macOS-recordreplay.so";
+process.env.RECORD_REPLAY_FRONTEND = process.env.INPUT_URL;
```
Comment (jasonLaster) [inline .github/actions/test/main.js:39]: wow - really?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -63,19 +54,20 @@ jobs:
   test0:
     name: Test stripe 0
     runs-on: macos-latest
-    needs: [build, download-browser, download-node, download-driver]
+    needs: [wait_for_vercel, download-browser, download-node, download-driver]
```
Comment (jasonLaster) [inline .github/workflows/test.yml:57]: 👍

**Review by jasonLaster — COMMENTED** [review]
Looking good

File: .github/actions/test/main.js (hunk 1 above)
Comment (ryanjduffy) [reply .github/actions/test/main.js:39]: just trying stuff ...

**Pull request closed without merging**